dip-0028.md

```diff
@@ -70,40 +68,40 @@ them with solutions that would be relatively easy to implement. As a result, we
   Lists](https://github.com/dashpay/dips/blob/master/dip-0004.md)
 * [DIP-0006: Long-Living Masternode
   Quorums](https://github.com/dashpay/dips/blob/master/dip-0006.md#building-the-set-of-deterministic-connections)
-* [High-Performance Masternodes
+* [Evolution Nodes
   FAQ](https://github.com/dashpay/FAQs/blob/master/Platform/platform-activation-hpmn-faq.md)
 
 ## Overview
 
-A High-Performance Masternode (HPMN) is like a regular masternode (MN) but with three key
+An Evolution Node (EvoNodes) is like a regular masternode (MN) but with three key
 differences:
 
-1. The collateral to own an HPMN is 4000 DASH as opposed to 1000 DASH for regular masternodes.
-2. The recommended specs for HPMNs are higher than those for regular masternodes.
-3. HPMNs serve Platform along with Core, while regular masternodes only serve Core.
+1. The collateral to own an EvoNode is 4000 DASH as opposed to 1000 DASH for regular masternodes.
+2. The recommended specs for EvoNodes are higher than those for regular masternodes.
+3. EvoNodes serve Platform along with Core, while regular masternodes only serve Core.
 
-HPMNs will provide all the services for Platform. Regular masternodes will not perform any Platform
-services. In Core quorums, HPMNs will act just like regular masternodes. In the DAO, they will have
+EvoNodes will provide all the services for Platform. Regular masternodes will not perform any Platform
+services. In Core quorums, EvoNodes will act just like regular masternodes. In the DAO, they will have
 4 times more voting power due to their collateral being 4 times greater.
 
-HPMNs will receive 100% of the fees generated from Platform and 37.5% of the masternode portion of
+EvoNodes will receive 100% of the fees generated from Platform and 37.5% of the masternode portion of
 Core block rewards. Regular MNs will receive the remaining 62.5% of the masternode portion of Core
 block rewards and 0% of Platform fees.
 
-Because the reward distribution percentages are fixed, the number of HPMNs is expected to stabilize
+Because the reward distribution percentages are fixed, the number of EvoNodes is expected to stabilize
 around a fixed number based on the total number of masternodes (considering the current number of
-~3850 Masternodes, ~450 HPMNs are expected). This is because if there are more than that fixed
-number of HPMNs, running a regular MN will be more profitable than running an HPMN, and hosts will
-convert their HPMNs into MNs. This also works in reverse. We call this mechanism the _yield
+~3850 Masternodes, ~450 EvoNodes are expected). This is because if there are more than that fixed
+number of EvoNodes, running a regular MN will be more profitable than running an EvoNodes, and hosts will
+convert their EvoNodes into MNs. This also works in reverse. We call this mechanism the _yield
```

Comment:
```suggestion
~3850 Masternodes, ~450 evonodes are expected). This is because if there are more than that fixed
number of evonodes, running a regular MN will be more profitable than running an evonodes, and hosts will
convert their evonodes into MNs. This also works in reverse. We call this mechanism the _yield
```